python/mp/effects/__init__.py

```diff
@@ -1,19 +1,23 @@
 #subpackage 'effects' for mp module (Mega Prayer)
-from mp.effects import effect
-from mp.effects import sine_wave
-from mp.effects import throb
-from mp.effects import bounce
-from mp.effects import set_color
-from mp.effects import casino
+#from mp.effects import effect
+#from mp.effects import sine_wave
+#from mp.effects import throb
+#from mp.effects import bounce
+#from mp.effects import set_color
+#from mp.effects import casino
```

Comment:
Yeah I've never actually looked deeply into this. I sorta just always leave a blank one. If there's something you don't know, I don't know either.

Maybe @KellyOgilvie can chime in here.